pkg/collector/metric/types/benchmark_test.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright 2023.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package types_test
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/sustainable-computing-io/kepler/pkg/collector/metric/types"
+)
+
+var seededRand *rand.Rand = rand.New(
+	rand.NewSource(time.Now().UnixNano()))
+
+// SetAggrStat
+func BenchmarkUInt64StatCollectionSetAggrStatMissed(b *testing.B) {
+	instance := types.UInt64StatCollection{
+		Stat: make(map[string]*types.UInt64Stat),
+	}
+	b.ReportAllocs()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		uuidStr, _ := uuid.NewRandom()
+		key := uuidStr.String()
```

Comment:
can you use a different random str generator instead of importing `uuid`?

Reply:
any suggestion?

Reply:
good catch, uuid takes a lots of time on benchmark testing!